lib/jsdom/browser/Window.js

```diff
@@ -198,8 +199,10 @@ function Window(options) {
     }
     return startTimer(window, setInterval, clearInterval, ++latestTimerId, fn, ms, args);
   };
+  this.requestAnimationFrame = fn => this.setTimeout(fn, 1000 / 60, performanceNow());
```

Comment:
The use of setTimeout and clearTimeout is not correct here. It means that calling setTimeout then requestAnimationFrame will cause the latter to return 1, not 0. Similarly, it means that using clearTimeout or clearInterval, we can clear animation frame callbacks. Neither of these is per spec. See https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#animation-frames and the "animation frame callback identifier" business.

It sounds like the test suite is lacking here, so we should add new to-upstream tests for those scenarios.

Reply:
Should I duplicate the code for `setTimeout` changing `latestTimerId`, and implement another `stopTimer`, to be it's own thing?

Reply:
Yeah, I guess so, or use some abstraction that allows determining which timer variable to update maybe?

Reply:
Updtaed